writeups/ContributingToWikiToLearn_SrijanAgarwal.md

```diff
@@ -0,0 +1,169 @@
+
+* Any content can be modified. This leads to a continuous development of the existing material, to this philosophy of *collaboration* and *communication* among thinkers.
+* Everything on WikiToLearn is freely accessible and downloadable. In this way, knowledge is made available to everyone.
+
+## How can I contribute?
+ 
+WikiToLearn is actively developed both on the technical and the content side. It's possible to contribute in several ways, regardless of the user's skills and experience.
+
+### Translation
+Are you fluent in (at least) two languages? 
+Translating on WikiToLearn is an opportunity to practice your translation skills, at the same time you will learn about new things, and you will contribute to the growth of knowledge in the world . The content does not need to be written from scratch, the translation into several languages is as important as the content itself, to guarantee its accessibility to everyone.
+
+If you want to help with translation you can join the [#translation](https://chat.wikitolearn.org) channel.
+
+### Web development
+WikiToLearn is a project based on MediaWiki, so we always need developers to fix bugs and develop new innovative programs. Web development is split into groups and every group has its maintainer. If you want to help with tech you can join the [#tech](https://chat.wikitolearn.org) channel.
+
+### Setting up local WikiToLearn instance with WikiToLearnHome
+
+#### Requirements
+* 64-Bit Computer (docker need a 64 bit system)
+* GNU/Linux with docker or Windows/OSX with virtualization compatability and enabled
+* Stable internet connection (download size can be up to 10 GB)
+
+This procedure may fail if you have no Internet access or if the connection is filtered by firewall – this could happen in public places such as universities, libraries, airports, etc.
+
+#### GNU/Linux installation
+##### Prerequisites
+WTL and WTLH have some dependecies:
+
+* any version
+    * curl
+    * rsync
+    * dirname
+    * realpath
+    * git
+    * pandoc
+    
+* specific version required
+    * docker ( >= 1.10.3)
+    * python3
+    
+##### Ubuntu/Debian prerequisites install
+```sudo apt-get install curl rsync coreutils realpath git python3 pandoc```
+
+##### Arch prerequisites install
+```sudo pacman -Sy curl rsync coreutils git python3 pandoc```
+
+`realpath` is in `AUR`
+
+#### Docker installation
+Docker is a quickly evolving technology, and it may be possible that the your distro official repository do not have yet the docker version requierd to run WTL. We thus recommend to install docker following the [official guide](https://docs.docker.com/engine/installation/).
```

Comment:
your **distro's** official repository do**es** not have the ~yet~ docker version **required** to run WTL *yet*.